templates/standard.adoc

```diff
@@ -0,0 +1,52 @@
+= Pomegranates
+Author Minetest-Docs and Contributors 
```

Comment:
Why do we need this? Git already conveniently stores this info for us.

Reply:
It's required for us to also have the revision number, which I think is still useful while reading the documentation.

Reply:
This will get moved into a header anyways here shortly anyways